source/index.txt

```diff
@@ -1,10 +1,61 @@
+.. _intellij-index:
+
 ===========================
-MongoDB for IntelliJ Plugin
+{+intellij-full+}
 ===========================
 
 .. default-domain:: mongodb
 
-Content here will be added in DOCSP-45086.
+.. meta:: 
+   :description: The MongoDB for IntelliJ Plugin is a tool designed to improve the development experience for Java developers working with IntelliJ IDEA.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The {+intellij-full+} is designed to improve the development experience 
+for Java developers working with IntelliJ IDEA. The {+intellij-short+} builds on 
+the existing IntelliJ Database Explorer to streamline the process of writing 
+and managing MongoDB queries directly within the IDE.
+
+You can use the {+intellij-short+} to:
+
+- Connect to your MongoDB instance and interact with your data. 
+
+- Build and test Java queries faster.  
+
+- Identify query and data modeling anti-patterns in your application code.
+
+Features 
+--------
+
+The {+intellij-short+} currently supports the following features:
+
+- :ref:`Autocomplete <intellij-autocomplete>` for database, collection, and 
+  document field names. 
+
+- :ref:`Type validation <intellij-type-validation>` of document fields.
+
+- :ref:`Missing index warnings <intellij-index-warning>` for Java queries.
+
+Get Started 
+-----------
+
+:ref:`intellij-install`
+  Install the {+intellij-short+} from the JetBrains Marketplace.
+
+:ref:`intellij-connect`
+  Connect to your MongoDB deployment.
+
+Submit Feedback 
+---------------
+
+To report a bugs, request a feature, or submit general feedback, open a 
+ticket in the `MongoDB for IntelliJ Plugin JIRA project <https://jira.mongodb.org/projects/INTELLIJ/issues/INTELLIJ-143?filter=allopenissues>`__.
```

Comment:
did we mean to put https://jira.mongodb.org/projects/INTELLIJ/issues/ instead?

Reply:
Ah good catch. Fixed.